src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
+  const today = date;
```

Comment:
I'm not sure, but it seems like a useless variable for me

Reply:
your truth, I don't know why I created it xd
thx